Documentation/git-commit.txt

```diff
@@ -177,7 +177,7 @@ The `-m` option is mutually exclusive with `-c`, `-C`, and `-F`.
 	sole parent commit is a mistake, and the command prevents you
 	from making such a commit.  This option bypasses the safety, and
 	is primarily for use by foreign SCM interface scripts. See the 
-	`commit.allowEmpty` configuration variable in linkgit:git-config[1].
+	`commit.allowEmpty` in linkgit:git-config[1].
```

Comment:
If you do that, you probably also want to remove "the" before `commit.allowEmpty`.